src/coreclr/src/debug/createdump/resource.h

```diff
@@ -0,0 +1,3 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Do you need this empty file? I do not think it should be needed.

Reply:
I'll remove it.  I copied what the dbgshim project was doing.  It has an empty resource.h too. Should I remove that one too?

Reply:
Yes, that would be nice.